src/main/webapp/script.js

```diff
@@ -173,6 +173,42 @@ function getGraphDisplay(graphNodes, graphEdges) {
     const node = evt.target;
     node.tip.show();
   });
+
+  let searchElement = document.getElementById('search');
+  searchElement.onchange = function() {
+    if (searchNode(cy, searchElement.value)) {
+      document.getElementById('searcherror').innerText = "";
+    } else {
+      document.getElementById('searcherror').innerText = "Node does not exist.";
```

Comment:
What happens if the user deletes what's in the search? (then the name is empty so the node isn't found, right?) But we don't want the "Node does not exist"?

Reply:
If there is nothing in the search, it will essentially fit all of the nodes on screen (so just reset) so the error message won't show.